crates/agent/src/publications.rs

```diff
@@ -30,8 +30,11 @@ pub struct PublicationResult {
     /// The state of any related live_specs, prior to the draft being published
     pub live: tables::LiveCatalog,
     /// The build specifications that were output from a successful publication.
-    /// Will be empty if the publication was not successful.
+    /// May be empty if the build failed prior to validation. Contains any validation
+    /// errors.
     pub built: tables::Validations,
+    /// Errors that occurred while running tests.
```

Comment:
supernit: tables::Errors instead of Vec ?

Reply:
yeah, agreed. Done